SpellCheck/pom.xml

```diff
@@ -9,14 +9,6 @@
 		<version>2.1.2-SNAPSHOT</version>
 	</parent>
 
-	<repositories>
-	    <!--other repositories if any-->
-	    <repository>
-	        <id>project.local</id>
-	        <name>project</name>
-	        <url>file:${project.basedir}/repo</url>
-	    </repository>
-	</repositories>
```

Comment:
We don't need this any more.  As a benefit, we shouldn't see `${project.basedir}` directories being created on our filesystems any more.